apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/FeatureExtractorSamples.kt

```diff
@@ -5,18 +5,538 @@ import ch.tutteli.atrium.api.verbs.expect
 import kotlin.test.Test
 
 class FeatureExtractorSamples {
+
     data class Person(
+        val name: String,
+        val age: Int
+    )
+
+    @Test
+    fun itsFeature() {
+        val person = Person(name = "John Smith", age = 25)
+
+        expect(person) {
+            its { age } toBeLessThan 30 toBeGreaterThan 20
+        }
+
+        fails {
+            expect(person) its { age } toBeLessThan 20 toBeGreaterThan 30
+            //                         |               | not evaluated/reported because `toBeLessThan` already fails
+            //                         |               | use an expectation-group block; for more details: https://github.com/robstoll/atrium#define-single-expectations-or-expectation-groups
+            //                         | fails
+        }
+    }
+
+//    @Test
+//    fun its() {
+//        val person = Person(name = "John Smith", age = 25)
+//
+//        expect(person) .its { age }
+//                .toBeGreaterThan(18) toBeLessThan(35) // subject within this expectation-group is of type Int
+//            // subject here is back to type Person
+//
+//        fails {
+//            expect(person)
+//                .its { age }
+//                    // introduces an expectation-group block
+//                    // all expectations are evaluated inside an expectation-group block; for more details:
+//                    // https://github.com/robstoll/atrium#define-single-expectations-or-expectation-groups
+//                    .toBeGreaterThan(40) toBeLessThan(50)    // still evaluated, use `.its.` if you want fail fast behaviour // TODO fix comment
+//        }
```

Comment:
in most places we place the link right after `fails {` (I fixed some outliers in main). Thus I suggest we use here as well (and in other places in this file):
```suggestion
        fails {
            // all expectations are evaluated inside an expectation-group block; for more details:
            // https://github.com/robstoll/atrium#define-single-expectations-or-an-expectation-group

            expect(person).its({ age }) {
                it toBeGreaterThan(40)
                it toBeLessThan(50)    // still evaluated, use `.its { ... }` if you want fail fast behaviour
            }
        }
```